docs/source/tutorials/first-plugin.rst

```diff
@@ -0,0 +1,52 @@
+=================
+Your first plugin
+=================
+
+Sopel's most interesting features come from its plugins, either published by
+Sopel's developers or by third-party developers, and you can write your own
+plugins. But where do you start?
+
+Here is a very short example of code for your first plugin that contains one
+and only one command::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Reply with Hello!"""
+        bot.reply('Hello!')
+
+You can put this code in a Python file, placed into your Sopel plugin
+directory, such as ``~/.sopel/plugins/myplugin.py``. Once this is done, you can
```

Comment:
I think some readers would benefit from explicitly stating here that the bot will load the files it finds in the plugins directory during start-up.

```suggestion
You can put this code in a Python file, placed into your plugins directory,
such as ``~/.sopel/plugins/myplugin.py``, and Sopel will load the plugin when
it scans the directory during start-up. Once this is done, you can
```

Reply:
No, too much details. Right now the tutorial tries to guide the reader to act now and to experiment rather than just reading stuff. It is not the time to make the reader understand how it works, it is the time to make the reader experiment. I want the reader to have the "wahou" effect of "this is easy!", not overload them with too much information.